### PR TITLE
Topbar replacement (w/LoginTopBar)

### PR DESCRIPTION
**Pull request by carlosascari**
Replaced topbar in `HomePage`, `ProfilePage` and `PublicGroup` components to use `LoginTopBar` component

### CODE REVIEW

File: frontend/src/containers/PublicGroup.js [hunk 1]
```diff
@@ -185,7 +185,7 @@ export class PublicGroup extends Component {
     const { group } = this.props;
     return (
       <div className='PublicGroup PublicGroup--inactive'>
-        <OnBoardingHeader />
+        <LoginTopBar />
```
Comment (asood123) [inline frontend/src/containers/PublicGroup.js]: Doesn't it have to updated in PublicGroupHero as well?

Comment (carlosascari) [inline frontend/src/containers/PublicGroup.js]: You are right I missed it
I've made the changes to that component as well

File: frontend/src/components/public_group/PublicGroupHero.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ export default class PublicGroupHero extends React.Component {
     return (
       <section className='PublicGroupHero relative px2 bg-black bg-cover white' style={{backgroundImage: `url(${collectiveBg})`}}>
         <div className='container relative center'>
-          <PublicTopBarV2 loginRedirectTo={`/${group.slug}`} className='pt3 absolute top-0 left-0 right-0' />
+          <LoginTopBar loginRedirectTo={`/${group.slug}`} classNamxe='pt3 absolute top-0 left-0 right-0' />
```
Comment (carlosascari) [inline frontend/src/components/public_group/PublicGroupHero.js]: It is no longer needed, it has been removed

File: frontend/src/containers/HomePage.js [hunk 3]
```diff
@@ -5,7 +5,12 @@ import 'numbro/dist/languages'
 
 import formatCurrency from '../lib/format_currency';
 
+<<<<<<< HEAD
```
Comment (carlosascari) [inline frontend/src/containers/HomePage.js]: @asood123 This is probably the reason the tests failed in CircleCI

**Pull request merged**